src/react/webgl-renderer.js

```diff
@@ -131,9 +131,52 @@ export default class WebGLRenderer extends React.Component {
     }
   }
 
+  // Gets current size of canvas in css (logical/window) coordinates
+  _getCSSSize(canvas) {
```

Comment:
`clientWidth` and `clientHeight` have perf impacts. Since we already know the canvas' `width` and `height` styles (set in the `render` method) we don't need the work around here.